File: ClientSide/src/app/pages/configuration/models/notificationAndSyncConfigurationModel.ts
export class NotificationAndSyncConfigurationModel {

    public dataSyncPeriodHours: number;
    public notificationsPeriodHours: number; 

    constructor(
        dataSyncPeriodHours: number,
        notificationsPeriodHours: number
    ) {
        this.dataSyncPeriodHours = dataSyncPeriodHours;
        this.notificationsPeriodHours = notificationsPeriodHours;
    }

}

File: ServerSide/SAMS.BusinessLogic/Models/Configurations/NotificationAndSyncConfigurationModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.Configurations
{
    public class NotificationAndSyncConfigurationModel
    {
        public int DataSyncPeriodHours { get; set; }
        public int NotificationsPeriodHours { get; set; }
    }
}
